sites/upsun/src/get-started/stacks/drupal.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Deploy Drupal on {{% vendor/name %}}
 sidebarTitle: Drupal
-sectionBefore: PHP
+#sectionBefore: PHP
 #layout: single
 weight: -67
```

Comment:
```suggestion
weight: -57
```

Reply:
this is how Kemi has it set. let's do this and then we can merge this one while we're finishing the magento guide.